api/include/opentelemetry/trace/span.h

```diff
@@ -43,8 +43,13 @@ struct StartSpanOptions
   core::SystemTimestamp start_system_time;
   core::SteadyTimestamp start_steady_time;
 
+  // Explicitely set the parent of a Span.
+  //
+  // This defaults to an invalid span context. In this case, the Span is
+  // automatically parented to the currently active span.
+  SpanContext parent = SpanContext::GetInvalid();
```

Comment:
Just wondering if it would be more optimized to have  `SpanContext::GetInvalid()` method returning a static instance of invalid context ( using c++11 local atomic static ). Can be part of this or separate PR.

Reply:
That would be more efficient if we'd return a pointer, but as we return by value here this involves a copy anyway, and we wouldn't gain much by having a static singleton.